modules/cnf-troubleshooting-common-ptp-operator-issues.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-troubleshooting-common-ptp-operator-issues_{context}"]
+= Troubleshooting common PTP Operator issues
+
+Troubleshoot common problems with the PTP Operator by performing the following steps.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator on a bare-metal cluster with hosts that support PTP.
+
+.Procedure
+
+. Check the operator and operands are successfully deployed in the cluster.
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ptp
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            READY   STATUS    RESTARTS   AGE
+linuxptp-daemon-5j265           3/3     Running   0          24h
+linuxptp-daemon-s72q2           3/3     Running   0          24h
+ptp-operator-6b8dcbf7f4-czff8   1/1     Running   0          10d
+----
++
+[NOTE]
+====
+When the event bus is enabled, the number of ready linuxptp-daemon pods is `3/3`. If the event bus is not enabled, `2/2` is displayed.
+====
+
+. Check that supported hardware is found in the cluster.
++
+[source,terminal]
+----
+$ oc -n openshift-ptp get nodeptpdevices.ptp.openshift.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  AGE
+control-plane-0.example.com           10d
+control-plane-1.example.com           10d
+compute-0.example.com                 10d
+compute-1.example.com                 10d
+compute-2.example.com                 10d
+----
+
+. Check the available PTP network interfaces:
++
+[source,terminal]
+----
+$ oc -n openshift-ptp get nodeptpdevices.ptp.openshift.io <node_name> -o yaml
+----
++
+where:
++
+<node_name>:: is the node you want to query, for example, `compute-0.example.com`.
++
+.Example output
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: NodePtpDevice
+metadata:
+  creationTimestamp: "2021-09-14T16:52:33Z"
+  generation: 1
+  name: compute-0.example.com
+  namespace: openshift-ptp
+  resourceVersion: "177400"
+  uid: 30413db0-4d8d-46da-9bef-737bacd548fd
+spec: {}
+status:
+  devices:
+  - name: eno1
+  - name: eno2
+  - name: eno3
+  - name: eno4
+  - name: enp5s0f0
+  - name: enp5s0f1
+----
+
+. Check that the PTP interface is successfully synchronized to the primary clock by accessing the `linuxptp-daemon` pod for the parent node.
+
+.. Remote shell into the required `linuxptp-daemon` container:
++
+[source,terminal]
+----
+$ oc -n openshift-ptp rsh -c linuxptp-daemon-container <linux_daemon_container>
+----
++
+where:
++
+<linux_daemon_container>:: is the container you want to diagnose, for example `linuxptp-daemon-5j265`.
+
+.. Run the following `pmc` command to check the sync status of the PTP device, for example `ptp4l`.
```

Comment:
Will a use know what this is? And, at this point, will the user already have the package that provides `pmc` installed? (If not, it should be a prereq, too.)

Reply:
No need to install or do any config here, it's installed and configured on the container already, and we've have just remote shelled into the container. But I will update with by rexpanding out the tool name, "PTP Management Client (`pmc`)".